docs/content/guides/running-dagster-locally.mdx

```diff
@@ -0,0 +1,86 @@
+
+---
+
+## Understanding the dagster dev command
+
+The `dagster dev` command launches [Dagit](/concepts/dagit/dagit) and the [Dagster daemon](/deployment/dagster-daemon), allowing you to start a full deployment of Dagster from the command line.
+
+This command should be run in a Python environment where the `dagster` and `dagit` packages are installed. **Once started, the process should be kept running.**
+
+---
+
+## Locating your code
+
+Before you can start developing, you need to tell Dagster how to find the Python code containing your assets and jobs. There are a few ways to do this, which are outlined in the tabs below.
+
+**Note**: If using an example Dagster project, or if you used the [`dagster` CLI to create a project](/getting-started/create-new-project#bootstrapping-a-new-project), you can run the `dagster dev` command in the same folder as the project to load the project code.
+
+<!-- Lives in /next/components/mdx/includes/dagster/DagsterDevTabs.mdx -->
+
+<DagsterDevTabs />
+
+## Configuration
+
+- [Run and asset storage](#asset-and-run-storage)
+- [Local instance](#local-instance)
+
+### Asset and run storage
+
+When running `dagster dev`, you may see log output that looks like this:
+
+```shell
+Using temporary directory /Users/rhendricks/tmpqs_fk8_5 for storage.
+```
+
+This indicates that any runs or materialized assets created during your session won't be persisted once the session ends. This can be useful when using Dagster for temporary local development or testing, when you don't care about the results being persisted.
+
+To designate a more permanent home for your runs and assets, you can set the `DAGSTER_HOME` environment variable to a folder on your filesystem. Dagster will then use the specified folder for storage on all subsequent runs of `dagster dev`.
+
+### Local instance
+
+You can optionally use a `dagster.yaml` file to configure your Dagster instance - for example, to configure [run concurrency limits](/deployment/run-coordinator#limiting-run-concurrency) or specify that runs should be stored in a [Postgres database](/deployment/dagster-instance#postgres-storage) instead of on the filesystem.
+
+If the `DAGSTER_HOME` environment variable is set, `dagster dev` will look for a `dagster.yaml` file in the `DAGSTER_HOME` folder. If `DAGSTER_HOME` is not set, `dagster dev` will look for that file from the folder where the command was run.
+
+For the full list of options that can be set in the `dagster.yaml` file, refer to the [Dagster instance documentation](/deployment/dagster-instance).
+
+---
+
+## Moving to production
+
+`dagster dev` is primarily useful for running Dagster for local development and testing, but isn't suitable for the demands of most production deployments. For example, in a production deployment, you might want to run multiple Dagit replicas, have zero downtime continuous deployment of your code, or set up your Dagster daemon to automatically restart if it crashes.
+
```

Comment:
For this reason I don't think we should have dagster dev in the "running Dagster as a service" guide - for real usage i think we want the two commands running separately?